device/core/src/interfaces.ts

```diff
@@ -129,6 +129,12 @@ export interface DeviceClientOptions extends X509 {
    */
   productInfo?: string;
 
+  /**
+   * !!Digital Twin Use Only!!
+   * String used in MQTT username settting the Digital Twin modelID.
```

Comment:
modelId

Reply:
Well if we're gonna get picky, there are three 't''s in setting

---
In reply to: [431961220](https://github.com/Azure/azure-iot-sdk-node/pull/809#discussion_r431961220) [](ancestors = 431961220)